### PR TITLE
Films 2014 from csv

### PR DESCRIPTION
**Pull request by davojta**
push data for 2014 films online

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,17 @@ test_db.sqlite3
 
 .tox
 .cache
+
+.idea/
+.ipynb_checkpoints/
+next.filmfest.by.iml
```
Comment (staaas) [inline .gitignore:12]: what's this for?

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,17 @@ test_db.sqlite3
...
+*.ipynb
+./*.json
```
Comment (staaas) [inline .gitignore]: and this?

File: .gitignore [hunk 3]
```diff
@@ -6,3 +6,17 @@ test_db.sqlite3
...
+
+scripts/films.json_to_next_raw_data/node_modules/*
+scripts/films.json_to_next_raw_data/private/*
+scripts/films.json_to_next_raw_data/secret.js
+
+tests/__pycache__/*
```
Comment (staaas) [inline .gitignore]: just `__pycache__` should be enough

File: .gitignore [hunk 4]
```diff
@@ -6,3 +6,17 @@ test_db.sqlite3
 
 .tox
 .cache
+
```
Comment (staaas) [inline .gitignore:9]: could you please group them by purpose?

File: scripts/readme.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# scripts for filmfest.by
+
+Here we aggregate scripts which helps to process Cinema Perpetuum Modile festival data for new webstite
```
Comment (staaas) [inline scripts/readme.md]: which help
website

**Comment by staaas** [comment]
this pr looks like a duplicate of #66, isn't it?

**Comment by davojta** [comment]
@nott this has #66 as base, because i reuse scripts and add some logic to it.

Do you think i can make one big pr for all 3 datasets and scripts?

File: .gitignore (hunk 1 above)
Comment (davojta) [reply .gitignore:12]: I just added idea special files to git ignore. And couple more specific files like ipython notebook ones

File: .gitignore (hunk 2 above)
Comment (davojta) [reply .gitignore]: It is about .json files on root level to avoid adding it by accident

**Comment by staaas** [comment]
> Do you think i can make one big pr for all 3 datasets and scripts?

go for it

File: .gitignore (hunk 2 above)
Comment (staaas) [reply .gitignore]: `*.json` should be enough then

File: .gitignore (hunk 1 above)
Comment (staaas) [reply .gitignore:12]: so could you please group them and add comments?

e.g.

```
# jupyter
*.ipynb
.ipynb_checkpoints/
# jetbrains ides
.idea
# etc
```

File: .gitignore (hunk 2 above)
Comment (davojta) [reply .gitignore]: *.json excludes all json files in the project

File: .gitignore [hunk 6]
```diff
@@ -3,6 +3,26 @@
...
+# exclude any ipython notebooks
+.ipynb_checkpoints/
+*.ipynb
+
+# exclude node files
+scripts/films.json_to_next_raw_data/node_modules/
+scripts/films.json_to_next_raw_data/private/
+scripts/films.json_to_next_raw_data/result/
+scripts/films.json_to_next_raw_data/secret.js
```
Comment (abitrolly) [inline .gitignore:22]: These are added to version control. Why exclude them?

File: .gitignore [hunk 7]
```diff
@@ -3,6 +3,26 @@
...
+__pycache__
 .tox
 .cache
+
+# exclude idea ide files
+.idea/
+next.filmfest.by.iml
+
+# exclude root json data files added by accedent
+/*.json
```
Comment (abitrolly) [inline .gitignore]: Just remove them, no need to ignore.

File: .gitignore [hunk 8]
```diff
@@ -3,6 +3,26 @@
...
+
+# exclude any ipython notebooks
+.ipynb_checkpoints/
+*.ipynb
```
Comment (abitrolly) [inline .gitignore:16]: Tell me why .ipynb are needed here?

File: scripts/readme.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+Here we aggregate scripts which help to process Cinema Perpetuum Modile festival data for new website
+ 
+# scripts list
+
+* jsoncheck.py - python script to check json in /data folder
+* jury-converter-2016/jury-converter.js - nodejs script to convert manually crafted json for jury member of 2016 to migration suitable format
+* scripts/films.json_to_next_raw_data/download_data.js - used to download submission.json from website and later usage
+* scripts/films.json_to_next_raw_data/clean_films_json_for_private_data.js - clean from submissions.json all private data ( email, address, phone number )
+* scripts/films.json_to_next_raw_data/next_films.js - convert submissions by their id using translation.json for migrate suitable format
```
Comment (abitrolly) [inline scripts/readme.md]: `scripts/submission_download.js`, `scripts/submission_filter.js` and `scripts/submission_process.js`

File: .gitignore (hunk 8 above)
Comment (davojta) [reply .gitignore:16]: just to avoid unneeded release draft notebooks

File: .gitignore (hunk 7 above)
Comment (davojta) [reply .gitignore]: why not to ignore?)

File: .gitignore (hunk 6 above)
Comment (davojta) [reply .gitignore:22]: @abitrolly can you list files which have been added to repo and proceed with that ignores ?
I exclude the folders which should not be added to the repo. The only public folder should.
The content of results folder should be copied to raw data folders

File: .gitignore (hunk 7 above)
Comment (davojta) [reply .gitignore]: @abitrolly  i have removed that to not confuse you )

File: scripts/readme.md (hunk 9 above)
Comment (davojta) [reply scripts/readme.md]: done )

File: .gitignore (hunk 6 above)
Comment (abitrolly) [reply .gitignore:22]: s/exclude node files/files generated when converting films data from/

Ok.

File: .gitignore (hunk 7 above)
Comment (abitrolly) [reply .gitignore]: s/exclude root json data files added by accedent/@davojta working snippets and notebooks/

File: .gitignore (hunk 8 above)
Comment (abitrolly) [reply .gitignore:16]: Group them, so people know who to ask about how are they used. )

File: scripts/readme.md (hunk 9 above)
Comment (abitrolly) [reply scripts/readme.md]: Did you forget `.gitignore`? ;)

**Review by abitrolly — APPROVED** [review]
LGTM

**Comment by abitrolly** [comment]
Comment for future - get human readable names for images, like 987_cuerdas.jpg - that way search engines can give links to our site.

File: .gitignore (hunk 8 above)
Comment (davojta) [reply .gitignore:16]: @abitrolly i will try to group them in next iterations )

**Pull request merged**